packages/components/FocusZone/src/FocusZone.types.ts

```diff
@@ -46,6 +44,10 @@ export interface FocusZoneProps {
   onFocus?: (e?: any) => void;
 }
 
+export interface NativeProps extends FocusZoneProps {
```

Comment:
extends Omit<FocusZoneProps, 'isCircularNavigation> { navigateAtEnd?: NavigateAtEnd }

....

expot interfaceFocusZoneSlotProps { root: NativeProps }

Reply:
Fixed in newest commit